src/main/webapp/app/exercises/shared/feedback/node/feedback-node.component.html

```diff
@@ -18,9 +18,7 @@ <h4 class="feedback-item__category">{{ feedbackItem.name }} {{ feedbackItem.titl
 <details *ngIf="feedbackItemGroup" class="feedback-item-group" [open]="feedbackItemGroup.open">
     <summary class="feedback-item-group__summary" [ngClass]="'bg-' + feedbackItemGroup.color">
         <div class="feedback-item-group__summary__content">
-            <h4 class="feedback-item__category">
-                {{ feedbackItemGroup.name | titlecase }}
-            </h4>
+            <h4 class="feedback-item__category">{{ feedbackItemGroup.name | titlecase }} ({{ feedbackItemGroup.members.length }})</h4>
```

Comment:
I'd propose using [Badges](https://getbootstrap.com/docs/5.3/components/badge).

Reply:
In general this seems like a good idea but I believe it's gonna get confusing as the background is already colored. I will try this out later and add a screenshot here

Reply:
`text-bg-light` or `text-bg-dark` maybe?

Reply:
![Screenshot_20230110_234530](https://user-images.githubusercontent.com/25489463/211678316-51048384-1386-4798-98fb-d4fb593de233.png)

Reply:
Fourth option looks reasonable.